examples/solids/index.rst

```diff
@@ -340,14 +340,64 @@ Carrying through the differentiation :math:numref:`strain-energy-grad` for the m
    For example, if :math:`e_{ij} \sim 10^{-8}`, then naive computation of :math:`\bm I_3 - \bm C^{-1}` and :math:`\log J` will have a relative accuracy of order :math:`10^{-8}` in double precision and no correct digits in single precision.
    When using the stable choices above, these quantities retain full :math:`\varepsilon_{\text{machine}}` relative accuracy.
 
+.. admonition:: Mooney-Rivlin model
+   :class: dropdown
```

Comment:
These blocks render as
![image](https://user-images.githubusercontent.com/3303/98632295-7ec63300-22dc-11eb-87ff-f46a92eda01b.png)
so we can include an adequate derivation without being disruptive to the narrative.